Fix minor typo

drdarshan opened the pull request:
The argument for compute target attach should be ``--type``  instead of ``-type``.

azurecla commented:
Hi __@ChrisCompy__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

drdarshan committed:
Fix minor typo

The argument for compute target attach should be ``--type``  instead of ``-type``.
